edalize/templates/design_compiler/design-compiler-read-sources.tcl.j2

```diff
@@ -0,0 +1,9 @@
+# Auto-generated project tcl file to read all sources
+
+{% if incdirs -%}
+set search_path [list {{ incdirs|join(' ') }}]
```

Comment:
I use fusesoc and I used this tool to generate DC analyze scripts. I need to add timing libraries to search path, but I don't want to mention those in Core file. (Timing libraries path may vary for different servers or users)
However, this line overwrites search_path and I can't set search_path in my scripts. (I can but it won't work :sweat_smile: )
I suggest to concatenate incdirs with previous value of search_path: 
```suggestion
set search_path [concat [list {{ incdirs|join(' ') }}] $search_path]
```
(I did test this solution and it worked for me)

Reply:
That sounds like a good idea. @davideschiavone, can you change that?